src/components/confirm-dialog/confirm-dialog.js

```diff
@@ -0,0 +1,43 @@
+import template from "./confirm-dialog.html";
+import { MDCDialog } from "@material/dialog";
+
+export class ConfirmDialogComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  showDialog() {
+    this.dialog.show();
+  }
+
+  onOkClick() {
+    console.log("accepted");
+  }
+
+  onCancelClick() {
+    console.log("declined");
```

Comment:
please expose this handler via props --so parent component will be able to subscribe to this event